cmd/lncli/commands.go

```diff
@@ -1790,6 +1790,10 @@ var getChanInfoCommand = cli.Command{
 			Name:  "chan_id",
 			Usage: "the 8-byte compact channel ID to query for",
 		},
+		cli.StringFlag{
+			Name:  "chan_point",
+			Usage: "the channel point in format txid:index",
```

Comment:
nit: mention it's optional if `chan_id` is used? Or add to both descriptions that only one of the flags is required.

Reply:
The `chan_id` takes precedence if both are set.